install/bootstrap_livecd_usb.md

```diff
@@ -682,38 +704,14 @@ On first login (over SSH or at local console) the LiveCD will prompt the adminis
    were downloaded to a system using the instructions in [Check for Latest Workarounds and Documentation Updates](../update_product_stream/index.md#workarounds). Use that set of RPMs rather than downloading again.
 
    ```bash
-   linux# wget https://artifactory.algol60.net/artifactory/csm-rpms/hpe/stable/sle-15sp2/docs-csm/1.2/noarch/docs-csm-latest.noarch.rpm
-   linux# wget https://storage.googleapis.com/csm-release-public/shasta-1.5/csm-install-workarounds/csm-install-workarounds-latest.noarch.rpm
-   linux# scp -p docs-csm-*rpm csm-install-workarounds-*rpm ncn-m001:/root
-   linux# ssh ncn-m001
-   pit# rpm -Uvh --force docs-csm-latest.noarch.rpm
-   pit# rpm -Uvh --force csm-install-workarounds-latest.noarch.rpm
-   ```
-
-1. Check the pit-release version.
-
-   ```bash
-   pit# cat /etc/pit-release
-   ```
-
-   Expected output looks similar to the following:
-
+   pit# rpm -Uvh --force https://storage.googleapis.com/csm-release-public/shasta-1.5/docs-csm/docs-csm-latest.noarch.rpm
+   pit# rpm -Uvh --force https://storage.googleapis.com/csm-release-public/shasta-1.5/csm-install-workarounds/csm-install-workarounds-latest.noarch.rpm
```

Comment:
Same here.  This is updated after we tell them to apply workarounds above.

Reply:
I just checked the release/1.2 branch of workarounds and I find two WARs there.  I don't think either of those should be done in 1.2.   We should get a call on whether we plan to keep using this repo in 1.2.